### PR TITLE
Validate PR 574

### PR DESCRIPTION
**Pull request by mapedraza**
Imported changes from PR #574 

Added a test that disconnect the DB and check if retrieving data gives error 500

### CODE REVIEW

**Comment by mapedraza** [comment]
Testing the case proposed by PR #574 is not working.

When requesting to the API with the DB disconnected, the following error is given:

```
(node:30203) [MONGODB DRIVER] Warning: the options [servers] is not supported
(node:30203) [MONGODB DRIVER] Warning: the options [caseTranslate] is not supported
(node:30203) [MONGODB DRIVER] Warning: the options [dbName] is not supported
        1) should return 500
```

File: test/unit/sthDatabase_test.js [hunk 1]
```diff
@@ -1205,4 +1264,5 @@ describe('sthDatabase tests', function() {
 
         describe('final clean up', cleanDatabaseTests);
     });
+    /*eslint-enable */
```
Comment (mapedraza) [inline test/unit/sthDatabase_test.js:1267]: This should be removed

File: test/unit/sthDatabase_test.js [hunk 2]
```diff
@@ -21,7 +21,7 @@
  * please contact with: [german.torodelvalle@telefonica.com]
  */
 
-/* eslint-disable consistent-return */
+/*eslint-disable */
```
Comment (mapedraza) [inline test/unit/sthDatabase_test.js:24]: This should be restored

**Comment by mapedraza** [comment]
In https://github.com/telefonicaid/fiware-sth-comet/pull/577/commits/36f486d69f87d35da21f40c97a1e253bb1d6d25e
Moved test from `test/unit/sthDatabase_test.js` to `test/unit/sth_test.js`

**Comment by Gauravp-NEC** [comment]
@mapedraza , Did this PR is in progress? Are you working on it?

**Comment by Gauravp-NEC** [comment]
@mapedraza @fgalan , This PR is overpassed by PR :https://github.com/telefonicaid/fiware-sth-comet/pull/583

**Comment by fgalan** [comment]
Having merged PR https://github.com/telefonicaid/fiware-sth-comet/pull/574 (and PR https://github.com/telefonicaid/fiware-sth-comet/pull/583) I think this one could be closed.

**Comment by mapedraza** [comment]
After merging https://github.com/telefonicaid/fiware-sth-comet/pull/583 this PR does not make sense.

**Pull request closed without merging**